### ISSUE TITLE
[BUG] SSD SMART Attributes do not match SmartCTL attribute names (170, 173, 174, 181, 183, 189, 202, 206)

### ISSUE THREAD

**Issue by brkirkpatrick** [AnalogJ/scrutiny#332]
Scrutiny incorrectly categories SMART ID 189 for one of my drives (M4 SSD) as "high fly writes." Smartctl reports this ID as "Factory_Bad_Block_Ct."

![scrutiny_id-189](https://user-images.githubusercontent.com/44349574/178385354-c857121f-9fdd-4e80-8b5d-59b345c2e90e.JPG)
![smartctl_id-189](https://user-images.githubusercontent.com/44349574/178385358-ab8315c1-b166-49ae-8691-90b31c454e12.JPG)

**Comment by AnalogJ** [comment]
Sigh, looks like the same issue as this one: https://bugzilla.redhat.com/show_bug.cgi?id=1140791

Basically:

> Anyway, the SMART attributes are unfortunately not standardized and the other SMART software, smartmontools, have a whole mechanism to report manufacturer/model specific parameter assignment.

**Comment by AnalogJ** [comment]
I guess I could do some tests of the SmartCTL provided `name` to determine the actual attribute to compare against. 

Unfortunately this is not really something that Scrutiny is currently designed to support, so it'll require some time to figure out.

**Comment by brkirkpatrick** [comment]
Would perhaps be useful if there was a way to ignore certain parameters as a temporary workaround? This SSD is the only drive showing as failed, and would be nice to make sure I notice if it _actually_ starts failing from another parameter.

**Comment by brkirkpatrick** [comment]
Arg, clicked wrong button and accidentally closed, now reopened. Sorry :)

**Comment by StephanRoemer** [comment]
> Would perhaps be useful if there was a way to ignore certain parameters as a temporary workaround? This SSD is the only drive showing as failed, and would be nice to make sure I notice if it _actually_ starts failing from another parameter.

Agreed, that would be a good way. I'm currently also experiencing this error message with an SSD.

**Comment by milouz1985** [comment]
Hi,

I'm facing the exact same problem. does anyone found a workaround?
I'm considering using collector.yaml to use a "patched" version of smartctl which would return an other id than 189 for my M4.

**Comment by kaysond** [comment]
This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.

**Comment by brkirkpatrick** [comment]
> This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.

Still relevant, continue to have drives showing as failed due to incorrectly interpreted SMART attributes.

**Comment by kaysond** [comment]
> > This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.
> 
> Still relevant, continue to have drives showing as failed due to incorrectly interpreted SMART attributes.

See https://github.com/AnalogJ/scrutiny/pull/1024
